src/screens/InviteRequiredScreen.tsx

```diff
@@ -1,91 +1,31 @@
 import { t } from 'i18next';
-import React, { useEffect } from 'react';
+import React from 'react';
 import { Text, View } from 'react-native';
 import {
-  ActivityIndicatorView,
   OAuthLogoutButton,
   OAuthLogoutButtonStyles,
   createStyles,
 } from '../components';
 import { tID } from '../common/testID';
-import {
-  useActiveAccount,
-  useActiveProject,
-  useConsent,
-  useStyles,
-} from '../hooks';
-import { LoggedInRootScreenProps } from '../navigators/types';
-
-export const InviteRequiredScreen = ({
-  navigation,
-}: LoggedInRootScreenProps<'InviteRequired'>) => {
-  const {
-    account,
-    isLoading: isLoadingAccount,
-    isFetched: isFetchedAccount,
-  } = useActiveAccount();
-  const {
-    activeProject,
-    isLoading: isLoadingProject,
-    isFetched: isFetchedProject,
-  } = useActiveProject();
-
-  const loadingProject = !isFetchedProject || isLoadingProject;
-  const loadingAccount = !isFetchedAccount || isLoadingAccount;
-  const loadingAccountOrProject = loadingProject || loadingAccount;
-  const hasAccountAndProject = !!(activeProject?.id && account?.id);
```

Comment:
While debugging, I found there to be too duplicated logic in here.  For example, this has the same issue for a user that has access to NO accounts (it will wait on `loadingProject` indefinitely).  Keeping it as simple as I've made it means that, once you reach this screen - unless RootStack reacts to something else - your _only_ option is to logout and start over.  This should be a rare edge case for a user to experience, so while we might want to improve that UX long-term, I like starting with simplicity for now.

Reply:
I did it to be consistent with how we had the screen prior but I agree that it doesn't add much value if this can be handled via closing the app and relaunching and it is already an edge case to begin with.